core/js/decanter.js

```diff
@@ -1,3 +1,434 @@
-/**
- * Empty until further notice.
- */
+// if NodeList doesn't support forEach, use Array's forEach()
+NodeList.prototype.forEach = NodeList.prototype.forEach || Array.prototype.forEach;
+
+//// TODO: move to components/main-nav.js file
```

Comment:
Remove todo comments. Create tickets instead.

Reply:
There's already a ticket for this: this will be done as part of the webpack config, which is next up as soon as this PR is approved. I added the TODO very early on so you would know this is just a temporary resting place for the code until webpack happens. I can delete it now if you like, but in general it helps me to have TODO's in the code.

Reply:
I removed this TODO